<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mydialog</class>
 <widget class="QDialog" name="mydialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="MyPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>MyPushButton</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>MyPushButton2</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>MyPushButton3</string>
   </property>
  </widget>
  <widget class="MyPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>MyPushButton4</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
